Gruntfile.js

```diff
@@ -192,19 +233,49 @@ module.exports = function(grunt) {
             dest: 'static/dist/xslt/'
           }
         ]
-      },
-      taskutils: {
-        files: [
-          {
-            expand: true,
-            flatten: true,
-            src: [ 'shared-libs/task-utils/src/task-utils.js' ],
-            dest: 'packages/task-utils/'
-          }
-        ]
       }
     },
     exec: {
+      cleanDdocs: {
+        cmd: 'rm -rf ddocs/medic/_attachments && mkdir ddocs/medic/_attachments'
+      },
+      packNodeModules: {
+        cmd: ['api', 'sentinel'].map(module => [
+              `cd ${module}`,
+              `npm --production install`,
+              `sed -i -e 's/"dependencies"/"bundledDependencies"/g' package.json`,
```

Comment:
Do you really need to do this? This feels much dodgier than it should be if it were written in 2018

Reply:
Maybe we should just specify bundledDependencies in the package.json and leave it there. We could have a build time check to ensure everything in dependencies is also in bundledDependencies. Then it would just work...